src/services/user/adminUnblockUser.js

```diff
@@ -1,12 +1,14 @@
 const {successResponse, errorResponse} = require('../../utils');
 const {adminUnblockUserProcess} = require('../../processes/scoring/admin-block-unblock-user');
+const UsersFunction = require('../../databases/functions/users');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `UsersFunction` module is imported but it's not clear if this module exports a `userUnblockByAdmin` function. Please ensure that the function is exported correctly from the module.



<!-- This is an auto-generated comment by OSS CodeRabbit -->